packages/inter-protocol/test/swingsetTests/assetReserve/bootstrap-assetReserve-upgrade.js

```diff
@@ -0,0 +1,254 @@
+// @ts-check
+
+import { Fail, NonNullish } from '@agoric/assert';
+import { AmountMath, makeIssuerKit } from '@agoric/ertp';
```

Comment:
is this file duplicated? on purpose?

Reply:
yes. no. (I renamed but an open buffer resaved the old path)